packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -118,14 +129,9 @@ export interface IMoveInfo {
 	movedSeq: number;
 
 	/**
-	 * All seqs at which this segment was moved. In the case of overlapping,
-	 * concurrent moves this array will contain multiple seqs.
-	 *
-	 * The seq at  `movedSeqs[i]` corresponds to the client id at `movedClientIds[i]`.
-	 *
-	 * The first element corresponds to the seq of the first move
+	 * Stores the seq, client id, and ref seq for each obliterate.
 	 */
-	movedSeqs: number[];
+	concurrentMoves: IConcurrentMoveInfo[];
```

Comment:
Do the properties here get mutated after initialization? Or can we make them `readonly`?

Reply:
Same as above - I don't think they can be made `readonly`.